src/components/Header/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
```

Comment:
I'm using `Link` from the routing package here just as an example, we will likely end up using `useHistory` to augment our own components.